### ISSUE TITLE
[Advice] Custom Planets

### ISSUE THREAD

**Issue by ChetwyndKid** [zmaster587/AdvancedRocketry#857]
Hi,
Sorry if this is already explained somewhere I might of missed it while I was reading. 

When you are creating a custom Giant Gas Planet, Does it need a DIMID? or is it not classed as a dimension as you cant visit it? 

Also if I wanted to add custom liquids to this Giant Gas Planet, How would I go about doing that? Take Minechem Helium for example. 

Another thing is, Is it possible to make the planet has Gas and Ore available for missions while its a Gas Giant? 
Thank you.

**Comment by zmaster587** [comment]
A dimid is not required for gas giants, however since right now the planet ID is the same as the dimid, it will be assigned a random planetID and may cause gaps in existing dimensions.

As for adding a custom fluid, you'd use the "gas" tag, however since forge only allows one type of fluid with the same name to exist at once, it can only be Minechem's Helium if either no other helium is registered, or it has its own fluid name.  Other than that it's just a matter of sticking the forge fluid name into the gas tag.

I don't think I'm going to add the ability to pull ores from gas giants because that doesn't really make sense, you should be able to launch asteroid mining missions from a ship orbiting a gas giant however.

**Comment by ChetwyndKid** [comment]
Yeah I understand the logic,  was just looking to create a planet that was
gas and I could retrieve things like ice from

**Comment by zmaster587** [comment]
Oh I see, I didn't think of anything like that.  Maybe even things like floating metallic dusts, etc.  I could possibly add chests as a thing to the unmanned drones, that way you could harvet ice, etc directly
